### PR TITLE
verify opinions on phone

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/ModelInfoPage.java [hunk 1]
```diff
@@ -57,4 +66,16 @@ public String readBattery() {
...
         return batteryInfoLabel.getText();
     }
+    
+    public boolean isOpenedModel(String brand, String model) {
+	String brandModel = brand + " " + model;
+	LOGGER.info("Title Model Page: " + titleModelPage.getText());
+	LOGGER.info("Value Brand and Model: " + brandModel);
+	return titleModelPage.getText().toUpperCase().equals(brandModel.toUpperCase());
+    }
+    public OpinionsTabPage isOpenOpinionsTabPage() {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/ModelInfoPage.java:76]: rename method

Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/ModelInfoPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	int value = comments.get(0).getRatingComment();
+	for (int i = 1; i < comments.size() - 1; i++) {
+	    if (!(value >= comments.get(i).getRatingComment())) {
+		LOGGER.error(value + "less than" + comments.get(i).getRatingComment());
+		return false;
+	    }
+	    value = comments.get(i).getRatingComment();
+	}
+	return true;
+    }
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java:48]: last element won't be verify

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    public boolean verifySortByDate() {
+	LocalDate date = comments.get(0).dateStringToLocalDate();
+	for (int i = 1; i < comments.size() - 1; i++) {
+	    if (!(date.compareTo(comments.get(i).dateStringToLocalDate()) >= 0)) {
+		LOGGER.error(date + "after than" + comments.get(i).dateStringToLocalDate());
+		return false;
+	    }
+	    date = comments.get(i).dateStringToLocalDate();
+	}
+	return true;
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java:59]: see above

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    public boolean verifyRateComment() {
+	int martTo = comments.get(0).getRatingComment();
+	comments.get(0).clickRateUp();
+	return comments.get(0).getRatingComment() > martTo;
+    }
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java:66]: Separate this method

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+    public boolean verifyUnRateComment() {
+	int martTo = comments.get(0).getRatingComment();
+	comments.get(0).clickRateDown();
+	return comments.get(0).getRatingComment() < martTo;
+    }
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java:72]: see above

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java (hunk 2 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java (hunk 3 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java (hunk 4 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java (hunk 5 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsTabPage.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**